### ISSUE TITLE
CDN

### ISSUE THREAD

**Issue by antfu** [wenyan-lang/cdn#1]
I create this repo for hosting files for CDNs. I am planning to use different branches for different projects.

jsDelivr has [an awesome feature](https://www.jsdelivr.com/?docs=gh) that direct host files from Github repos, and surprisingly it's kinda fast in China.

So I moved files that originally hosted by Netlify to this repo by [a Github Action](https://github.com/wenyan-lang/wenyan/blob/master/.github/workflows/cdn.yml):

`https://wy-lang.org/dist/core.js` (somehow it's very slow in China)

now becomes:

`https://cdn.jsdelivr.net/gh/wenyan-lang/cdn@wenyan-core/core.min.js` (and with the `.min.js` suffix, jsDelivr will automatically minimize for you) 

I also changed the CDN for IDE in https://github.com/wenyan-lang/ide/commit/3f03542d94022f627d3ca8e515f7d21a48f57940

So far it's just a quick experiment and I think it works well. In the future, you may consider moving fonts for wenyan-book by these steps to get better loading speed for users in China. :)

**Comment by LingDong-** [comment]
Very cool! 👍

Is it purely out of pursuit of neatness that we move all wanted files to this repo instead of just using their own jsDelivr links? e.g.:

https://cdn.jsdelivr.net/gh/LingDong-/wenyan-book/assets/font.woff2

**Comment by antfu** [comment]
Yeah, I think the font subset is ok to be in the repo and use is directly by jsDelivr. (Please also aware that 
jsDelivr only accept repo that smaller than 30MB)

The main reason this repo gets introduced is that [files like those](https://github.com/wenyan-lang/cdn/tree/wenyan-core) are CI/CD on every commit automatically. And for most of the time, you don't want to commit these files into source control and manually rebuild it repeatedly.

And yeah, maybe another way is to commit files into a different branch of the source repo instead this centralized repo.

**Comment by antfu** [comment]
FYI, I have moved the dist files to the [`cdn` branch](https://github.com/wenyan-lang/wenyan/tree/cdn) of the main repo. I think this repo becomes redundant, will delete it in some days (to keep the old link available for caches to update).

**Comment by LingDong-** [comment]
👍 Sounds good. Thanks!
